akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -186,7 +186,7 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
   /**
    * Scala API: Register a named entity type by defining the [[akka.actor.Props]] of the entity actor
    * and functions to extract entity and shard identifier from messages. The [[ShardRegion]] actor
-   * for this type can later be retrieved with the [[#shardRegion]] method.
+   * for this type can later be retrieved with the [[shardRegion]] method.
```

Comment:
I know IntelliJ is complaining about these, but are they correct? Is that something new in ScalaDoc? It finds whatever it is? Does it work with genjavadoc?

Reply:
I did google around for what scala doc supports and didn't find much: https://stackoverflow.com/questions/34491342/in-scaladoc-how-to-link-to-a-method-in-the-same-class

Without the # and step to in intellij works hence why i change some of them if i end up wanting to navigate there